### DISCUSSION TITLE
Successful code pushed to repository...but

### DISCUSSION THREAD

**Discussion by piapis** [community/community#23936] [Other Feature Feedback, Questions, & Ideas]
<p>As part of my submission to the coursera course, i git pushed this code to the repository: coursera-test/module2-solution/module2 solution. my problem is the code does not execute as a web page, but when i tested it in my local reposiitory, it worked just fine :frowning:</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>yup, it’s not automatic, you need to go to Settings and go to GitHub Pages tab, then turn it as a page,</p>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/xdvrx1/github-pages-tutorial" target="_blank" rel="noopener nofollow ugc">github.com</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/47092464/181080306-c905d54f-f3fc-4eed-9fdf-0aa916ae4a4e.jpeg" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/9/e/9edafd090e622946642a431eac73b19b745a0277_2_690x345.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/9/e/9edafd090e622946642a431eac73b19b745a0277_2_1035x517.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/3X/9/e/9edafd090e622946642a431eac73b19b745a0277.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/9/e/9edafd090e622946642a431eac73b19b745a0277_2_10x10.png"></div>

<h3><a href="https://github.com/xdvrx1/github-pages-tutorial" target="_blank" rel="noopener nofollow ugc">xdvrx1/github-pages-tutorial</a></h3>


  <p><span class="label1">GitHub Pages will enable you to simply turn your GitHub repo into a static website</span></p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by piapis** [comment]
<p>thank you for your advise</p>
